docker-compose.yml

```diff
@@ -0,0 +1,71 @@
+services:
+  flask:
+    build:
+      context: .
+      dockerfile: Dockerfile
+      target: runner-dev
+    ports:
+      - "8000:8000"
+      - "5055:5055"
+    depends_on:
+      - mongo
+      - redis
+    volumes:
+      - .:/usr/src/app
+    environment:
+      - MONGO_URL=${MONGO_URL}
```

Comment:
For mongo and redis URLS, I presume we'd want to us ethe `mongo` and `redis` services by default, so could set defaults as
```
MONGO_URL=${MONGO_URL:-<container_dsb_equivalent}
REDIS_UR:=${REDIS_URL:-<container_dsn_equivalent}
```